classes/passwd.oeclass

```diff
@@ -19,18 +19,14 @@ GROUP_FILES ?= "${SRCDIR}/group"
 PASSWD_POSTFUNC:machine = " do_install_passwd"
 do_install[postfuncs] += "${PASSWD_POSTFUNC}"
 do_install_passwd () {
-	i=0
 	for f in ${PASSWD_FILES} ; do
-		mkdir -p ${D}/${passwddir}
-		let i=$i+1
-		cp $f ${D}/${passwddir}/${PN}.$i
+		mkdir -p ${D}/${passwddir}/${PN}
+		cp $f ${D}/${passwddir}/${PN}/
```

Comment:
This would not end very well with something like

`PASSWD_FILES="${SRCDIR}/foo/my-file ${SRCDIR}/bar/my-file`